packages/expo-maps/src/google/GoogleMapsView.tsx

```diff
@@ -2,9 +2,13 @@ import { requireNativeView } from 'expo';
 import * as React from 'react';
 import { Platform } from 'react-native';
 
-import type { MapProps } from './GoogleMaps.types';
+import type { MapViewType, MapProps, SetCameraPositionConfig } from './GoogleMaps.types';
 
-let NativeView: React.ComponentType<MapProps> | null = null;
+type NativeMapViewProps = MapProps & {
+  ref: React.MutableRefObject<MapViewType | null>;
```

Comment:
```suggestion
  ref?: Ref<MapViewType>;
```
Can't we add this to the `MapProps`?